.github/workflows/grafana.yml

```diff
@@ -9,9 +9,13 @@ env:
 
 on:
   workflow_dispatch:
+  schedule:
+    # Run every Wednesday at 8am
+    - cron: "0 8 * * 3"
   push:
     paths:
-      - 'grafana/*'
+      - 'grafana/**'
+      - '!grafana/README.md'
```

Comment:
what I think I observed was that this workflow would run as expected when there were changes to `'grafana/*'` in a branch that was not main (e.g. a PR). But would not run when the branch was merged to main when there was no merge commit - my suspicion is that there is no `push` event as when the commit merges cleanly it reuses the commit from the PR. This reduces duplication for running tests, but means that deploys don't happen when we want them to. 

As such, my suspicion is that changing the wildcards here will not make a difference to the issue that we've been having.

Reply:
Oh of course, that would explain why it wasn't running on merge to main. I wasn't sure how the `push/paths` and `schedule` options would work together anyway, but having read your explanation I'll remove the `push/paths` and just have this run on a schedule in future.